README.md

```diff
@@ -10,118 +10,26 @@ so users can purchase [proof-of-stake tickets](https://docs.decred.org/mining/pr
 on the [Decred](https://decred.org/) network and have the pool of wallet servers
 vote on their behalf when the ticket is selected.
 
+**NB:** In late 2018, [a proposal](https://proposals.decred.org/proposals/522652954ea7998f3fca95b9c4ca8907820eb785877dcf7fba92307131818c75)
+was approved by stakeholders to rename "Stakepool" to "Voting Service Provider", a.k.a. "VSP".
+These names are used interchangably in this repository.
+
 ## Architecture
 
-![Voting Service Architecture](img/architecture.png)
+![Voting Service Architecture](docs/img/architecture.png)
 
 - It is highly recommended to use at least 2 dcrd+dcrwallet+stakepoold nodes for
   production use on mainnet.
```

Comment:
Would a link to https://docs.decred.org/advanced/operating-a-vsp/ be beneficial here?

Reply:
Good idea